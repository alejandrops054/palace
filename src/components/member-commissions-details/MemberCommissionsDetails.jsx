import React from 'react'
import {Input} from '../index'


const MemberCommissionsDetails = ()=> {
    return (
        <div>
            <div className="card">
                <div className="card-header">
                    <div className="card-title">
                        <h3>Member Commissions Details</h3>
                    </div>
                </div>
                <div className="card-body">
                    <div className="row">
                        <div className="col-md-8">
                        <Input type="text" label="Commission Down Payment Required" col={12} icon='fas fa-percentage' placeholder='10.00' />
                        </div>
                        <div className="col-md-4">
                        <Input type="text" label="   " col={12} icon='fas fa-dollar-sign' placeholder='410.00' />
                        </div>
                    </div>
                    <div className="row ">
                        <div className="col-md-6  ">
                        <Input type="text" label="Commission Total Amount" col={12} icon='fas fa-dollar-sign' placeholder='4,100.00' />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    )
}



export default MemberCommissionsDetails
