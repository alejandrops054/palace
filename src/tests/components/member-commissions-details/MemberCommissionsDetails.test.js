/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react'
import {shallow} from 'enzyme'

import MemberCommissionsDetails from '../../../components/member-commissions-details/MemberCommissionsDetails'

describe('Pruebas del componente MemberCommissionsDetails', () => {
    test('Creacion de MemberCommissionsDetails', () => {
        
        const wrapper = shallow(<MemberCommissionsDetails/>)

        expect(wrapper).toMatchSnapshot();
    })
    
})
