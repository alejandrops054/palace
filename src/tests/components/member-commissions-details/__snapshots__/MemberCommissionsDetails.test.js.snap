// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente MemberCommissionsDetails Creacion de MemberCommissionsDetails 1`] = `
<div>
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      <div
        className="card-title"
      >
        <h3>
          Member Commissions Details
        </h3>
      </div>
    </div>
    <div
      className="card-body"
    >
      <div
        className="row"
      >
        <div
          className="col-md-8"
        >
          <Input
            col={12}
            icon="fas fa-percentage"
            label="Commission Down Payment Required"
            placeholder="10.00"
            type="text"
          />
        </div>
        <div
          className="col-md-4"
        >
          <Input
            col={12}
            icon="fas fa-dollar-sign"
            label="   "
            placeholder="410.00"
            type="text"
          />
        </div>
      </div>
      <div
        className="row "
      >
        <div
          className="col-md-6  "
        >
          <Input
            col={12}
            icon="fas fa-dollar-sign"
            label="Commission Total Amount"
            placeholder="4,100.00"
            type="text"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
